frontend/src/modules/activity/components/activity-form-drawer.vue

```diff
@@ -86,14 +86,15 @@
             @blur="$v.activityType.$touch"
             @change="$v.activityType.$touch"
           >
-            <div
+            <el-option
+              :value="null"
               class="px-3 py-2.5 text-brand-500 text-xs leading-5 transition hover:bg-gray-50 cursor-pointer"
               @click="emit('add-activity-type')"
             >
               + Add activity type
-            </div>
+            </el-option>
             <div
-              v-if="Object.keys(types.custom).length > 0"
+              v-if="Object.keys(types.custom.other).length > 0"
```

Comment:
Hey! Could you please remove `other`? We are listing all `custom` types
Also this breaks the dropdown if the user has never created any custom activity type, because `other` won't exist

Reply:
Actually when a user has never created a custom activity then its `types.custom` object would look something like `{}`
So calling `types.custom.other` breaks the dropdown
But when a user has created some custom activity then deleted it then its types.custom will look like this { "other": {} } 
So if we calculate length of `types.custom` then it will 1 because `types.custom = { "other": {} } `
and hence `Object.keys(types.custom).length > 0` will become true and hence Custom will be shown even if there are no custom events 
![Screenshot from 2023-06-10 01-26-45](https://github.com/CrowdDotDev/crowd.dev/assets/67850763/1286591b-7746-4ba9-926a-634ce6fa0071)
So adding a ternary opeator : `types.custom.other? Object.keys(types.custom.other).length > 0 : Object.keys(types.custom).length > 0`